problems/isOdd.js

```diff
@@ -7,8 +7,15 @@
 
 */
 
-function isOdd() {
-
+function isOdd(n) {
+ if (n % 2 !== 0) {
```

Comment:
Since this conditional will return true if the number is odd and false if it is even, consider an implicit return instead of an if-else statement.